README.md

```diff
@@ -0,0 +1,32 @@
+# Hello Microverse
+
+This project,  will set up a "Hello world" repository. No complex coding is required for this project. The main  goal here is to master all of the tools and best practices which we  will use  in all Microverse projects.
+
+# Built With
+* html5 & CSS3
+* linters
+* git & github 
+
+# Live Demo
+
+
+# Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+## Prerequisites: html5 , css3 , git 
+## Setup : clone this repistory using git clone command line .
+
```

Comment:
- [ ] [OPTIONAL] For the next time try to add more instructions about how to set up your project in any local environment, check [This Example](https://github.com/DanteAlonsoHT/Leaderboard_API) to let you know what we expect from students 😉